CODE_OF_CONDUCT.md

```diff
@@ -1,76 +1,25 @@
 # Contributor Covenant Code of Conduct
 
-## Our Pledge
+## TL;DR
```

Comment:
Why are we changing the CoC? Especially in this PR...

Reply:
My bad I'll revert